income-subset.csv

```diff
@@ -0,0 +1,60 @@
+age,education,marital-status,occupation,race,sex,native-country,income

```

Comment:
I don't think that y'all need to push any of this data or any of your BIN files to master since we either have copies or it's going to clutter up master.